app/controllers/api/rdvi/invitations_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class Api::Rdvi::InvitationsController < Api::V1::AgentAuthBaseController
+  def available_creneaux_count
+    total_creneaux = calculate_total_creneaux
+    render json: { available_creneaux_count: total_creneaux }
```

Comment:
on peut faire encore moins d'indirections
```suggestion
    render json: { available_creneaux_count: calculate_total_creneaux }
```